### PR TITLE
TD-943 - Super Admin - Administrators - Set up basic page

### PR DESCRIPTION
**Pull request by Auldrin-Possa**
**JIRA link**
https://hee-tis.atlassian.net/browse/TD-943

**Description**
Super Admin - Administrators list page added.

**Screenshots**
Updated in the Jira ticket.

-----
**Developer checks**

I have:
- [x] Run the formatter and made sure there are no IDE errors 
- [x] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript
- [x] Tested any Views or partials created or changed with [Wave Chrome plugin]
- [ ] Updated/added documentation in [Confluence]
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [x] Scanned over my pull request in GitHub and addressed any warnings from the GitHub Build and Test checks.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/SuperAdmin/AdminAccounts/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                    }
+                </div>
+                <nav class="side-nav-menu-bottom" aria-label="Bottom navigation bar">
+                    <partial name="~/Views/SuperAdmin/Shared/_UserAccountsSideNavMenu.cshtml" model="Model.CurrentPage" />
+                </nav>
+            </div>
+        </div>
+    </div>
+</div>
+}
```
Comment (kevwhitt-hee) [inline DigitalLearningSolutions.Web/Views/SuperAdmin/AdminAccounts/Index.cshtml:123]: This } is appearing in the page and probably needs removing.

File: DigitalLearningSolutions.Web/Views/SuperAdmin/AdminAccounts/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                    {
+                        <partial name="SearchablePage/_TopPagination" model="Model" />
+                        <partial name="SearchablePage/_ResultCountAndPageAlerts" model="Model" />
+                        <div id="searchable-elements">
+                            @foreach (var admin in Model.Admins)
+                            {
+                                <partial name="_SearchableAdminAccountsCard" model="admin" />
+                            }
+                        </div>
+                        <partial name="SearchablePage/_BottomPagination" model="Model" />
```
Comment (kevwhitt-hee) [inline DigitalLearningSolutions.Web/Views/SuperAdmin/AdminAccounts/Index.cshtml:112]: When Next page link is clicked, the applied filters are removed. Chhose a filter item -> click Search -> click Next : Removes filter and moves to page 2

Comment (Auldrin-Possa) [reply DigitalLearningSolutions.Web/Views/SuperAdmin/AdminAccounts/Index.cshtml:112]: filters are maintained on the next button click.

**Comment by Auldrin-Possa** [comment]
> Could we also tweak the display name for the result expander cards to show first name first as per the User Accounts page, i.e. "Kevin Whittaker" rather than "Whittaker, Kevin", please.

-Done

**Comment by kevwhitt-hee** [comment]
This is looking good. Two more tiny observations:

- Testers have fed back that the "Items per page" control should go above the paginated list to be consistent with other parts of the app.
- The "Export to Excel"button should have secondary/grey button styling rather than primary/green.

**Pull request closed without merging**